manifests/init.pp

```diff
@@ -228,7 +252,7 @@
   Stdlib::Absolutepath $service_logs_dir              = $rundeck::params::service_logs_dir,
   String $service_name                                = $rundeck::params::service_name,
   Optional[String] $service_script                    = undef,
-  String $service_ensure                              = $rundeck::params::service_ensure,
+  Enum['stopped', 'running'] $service_ensure          = $rundeck::params::service_ensure,
```

Comment:
Seems like Stdlib::Ensure::Service isn't in 4.21 even yet, let alone all our supported versions.

Reply:
With regard to service_ensure, docs say: "Valid values are stopped (also called false), running (also called true).", but recently merged https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/ensure/service.pp doesn't seem to account for true or false. I um, am not sure what to do.